### DISCUSSION TITLE
Proposta di aggiustamenti testo Annotazioni Automatiche.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#961] [Miglioramenti] [answered]
Buongiorno,

riportiamo una segnalazione dal Comune di Firenze.

A scopo di un miglioramento della leggibilità in italiano, proponiamo dei piccoli aggiustamenti, alle formule utilizzate per la generazione di alcune delle Annotazioni Automatiche. Di seguito il dettaglio, con la proposta:

Annotazione di nascita, a seguito di un evento di 'Figlio minore di chi ha acquista la cittadinanza italiana' (idUsecase:52143 - Citt_004)
"Con atto del ... (indicare la date dell'atto) n. ......... il ...........(indicare  l'Autorità che ha rilasciato l'Attestazione) ha attestato che .......... (nome e cognome) è (o: non è) cittadino(a)  italiano(a).";

Annotazione di nascita, a seguito di un evento di 'Trascrizione del decreto per matrimonio-unione civile-naturalizzazione-presentazione cittadino' (idUsecase:52153 - Citt_045)
"A seguito di Decreto del (Autorità che emana il Decreto) n. …. redatto in data …... (nome e cognome) ha acquistato la cittadinanza italiana.";

Annotazione di nascita, a seguito di un evento di 'Straniero nato in Italia che vi abbia risieduto legalmente senza interruzioni fino alla maggiore età' (idUsecase:51103 - Citt_011)
"(nome cognome)  in data ..... ha reso dinanzi all'Ufficiale di Stato Civile del Comune di Firenze, dichiarazione con la quale intende acquistare la cittadinanza italiana ai sensi dell'art. 4 c. 2 Legge 91/1992. La dichiarazione fu iscritta nei registri di cittadinanza in data 23/05/2024 con numero..... ";

Annotazione di nascita, a seguito di un evento di 'Straniero nato in Italia che vi abbia risieduto legalmente senza interruzioni fino alla maggiore età' (idUsecase:52122 - Citt_012)
"Con atto del ... (indicare la date dell'atto) n. ......... il ...........(indicare  l'Autorità che ha rilasciato l'Accertamento) ha accertato che .......... (nome e cognome) è (o: non è) cittadino(a)  italiano(a).".

In attesa di riscontro,

cordiali saluti.

**Comment by asqueglia** [comment]
Buongiorno, verifichiamo rispetto al formulario vigente e valuteremo le vostre proposte di miglioramento.

**Comment by asqueglia** [comment, marked as answer]
Buongiorno con il rilascio della patch 1.33.3, la segnalazione è stata corretta. Chiudiamo pertanto la discussione
